Android.bp

```diff
@@ -63,6 +63,29 @@ cc_library {
     },
 }
 
+cc_library {
+    name: "libsndcardparser",
+    vendor: true,
+    srcs: ["examples/sndcardparser/sample_sndcardparser.c"],
+    cflags: ["-Werror"],
+}
+
+cc_library {
+    name: "libsample_pcm_plugin",
+    vendor: true,
+    srcs: ["examples/plugins/sample_pcm_plugin.c"],
+    cflags: ["-Werror", "-Wno-unused-parameter"],
+    local_include_dirs: ["include"],
+}
+
+cc_library {
+    name: "libsample_mixer_plugin",
+    vendor: true,
+    srcs: ["examples/plugins/sample_mixer_plugin.c"],
+    cflags: ["-Werror", "-Wno-unused-parameter"],
+    local_include_dirs: ["include"],
+}
+
```

Comment:
Thanks @dvdli for review. Will move makefile changes to examples/plugins/Android.bp. Just to inform, for this I will have to export tinyalsa headers as header_libs so that plugins can use the header file present in include directory

Reply:
done